### PR TITLE
Feature/ele 607 create calendar view

### PR DESCRIPTION
**Pull request by Victor-Lindh**

### CODE REVIEW

File: src/views/Overviews/EventsOverview/EventsHeader/Datepicker.tsx
Comment (dannelundqvist) [inline src/views/Overviews/EventsOverview/EventsHeader/Datepicker.tsx:1]: Seems very generic, would be a good candidate to refactor into a generic component to be used elsewhere as well.

**Review by dannelundqvist — CHANGES_REQUESTED** [review]
A lot of ground covered. Fantastic job! Nicely structured and easy to read the code.

1. I marked out a few easy wins for refactoring to generic components. We could do it now if you think we have time, otherwise later.

2. Generally we are renaming sectors to sections. (Was hard for you to know, sorry, we should have told you). This would be good to fix now so we don't add more debt on the pile.

3. I am not really fond of putting the planning and calendar overview in a sub directory to views. They have no strong connection to each other logically or structurally. Planning detail have more in common with planning overview and calendar overview with calendar event.
I want really want us to keep the view structure completely flat as each view should be seen as independent and should be possible to open in many different situations.

File: src/views/Overviews/PlanningOverview/PlanningTable/Columns/Newsvalue.tsx
[the remark this answers is not in the document]
Comment (Victor-Lindh) [reply src/views/Overviews/PlanningOverview/PlanningTable/Columns/Newsvalue.tsx:1]: Yes, I agree. In the process of doing some more refactoring of shared components.

**Comment by dannelundqvist** [comment]
> @dannelundqvist - Regarding:
> >    I really want us to keep the view structure completely flat as each view should be seen as independent and should be possible to open in many different situations.
> 
> Would it be more fitting to place an `overviews` directory in the `src/` root?

Not really. They still are separate views and should live in `src/views/` as a flat folder structure:
```
views/
   Editor/
   Error/
   Event/
   EventsOverview/
   PlanningOverview/
   Planning/
```

File: src/views/Overviews/EventsOverview/EventsTable/Columns/Sector.tsx
[the remark this answers is not in the document]
Comment (Victor-Lindh) [reply src/views/Overviews/EventsOverview/EventsTable/Columns/Sector.tsx:1]: Fixed in new commit

File: src/views/Overviews/PlanningOverview/PlanningTable/Columns/Newsvalue.tsx
[the remark this answers is not in the document]
Comment (Victor-Lindh) [reply src/views/Overviews/PlanningOverview/PlanningTable/Columns/Newsvalue.tsx:1]: Committed and pushed.
Also, I went ahead and refactored some other components that were very generic as well.

File: src/views/Overviews/PlanningOverview/PlanningTable/Columns/Sector.tsx
[the remark this answers is not in the document]
Comment (Victor-Lindh) [reply src/views/Overviews/PlanningOverview/PlanningTable/Columns/Sector.tsx:1]: Fixed in new commit

**Comment by Victor-Lindh** [comment]
Pushed new commits with the above suggestions, plus some extra refactoring.

File: src/views/Overviews/EventsOverview/EventsHeader/Datepicker.tsx
Comment (Victor-Lindh) [reply src/views/Overviews/EventsOverview/EventsHeader/Datepicker.tsx:1]: Fixed in new commit

**Review by dannelundqvist — APPROVED** [review]
Looks a lot better now! 👍

**Pull request merged**